package.json

```diff
@@ -2,6 +2,11 @@
   "name": "@openedx/brand-openedx",
   "version": "1.0.0-semantically-released",
   "description": "The default branding and SASS theme package containing for Open edX applications. This package is designed to be copied and customized.",
+  "scripts": {
+    "build-tokens": "paragon build-tokens --source ./paragon/tokens/ --build-dir ./paragon/build -t dark -t light",
+    "build-dist-files": "rm -rf dist && mkdir dist && paragon build-scss --corePath ./paragon/core.scss --themesPath ./paragon/build/themes --source",
+    "pgn": "paragon help"
```

Comment:
[consider] Might rename `pgn` to `paragon:help` to be more explicit that this is a help command related to paragon.